Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -141,6 +148,33 @@ click: function () { closeClicked.raiseEvent(this); }');
 
     defineProperties(InfoBox.prototype, {
         /**
+         * A flag that allows scripts to run inside the infobox from the string.
+         * Due to security issues default is false, and it must be explicitly set after infobox creation.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         *
+         * @example
+         * var viewer = new Cesium.Viewer('cesiumContainer');
+         * viewer.infoBox.allowScripts = true; // now you can set entity descriptions with <script> tags to be parsed
```

Comment:
I don't think we need this additional comment in the example, instead just make sure your documentation above makes this clear.